pkg/model/components/kubecontrollermanager.go

```diff
@@ -120,7 +121,7 @@ func (b *KubeControllerManagerOptionsBuilder) BuildOptions(o interface{}) error
 
 	kcm.LogLevel = 2
 
-	image, err := Image("kube-controller-manager", clusterSpec)
+	image, err := assets.Image("kube-controller-manager", clusterSpec)
```

Comment:
This will have to go through the assetbuilder instance in our Context, but yes, I'd like to get this commit in...